ibm/table_ibm_cis_domain.go

```diff
@@ -0,0 +1,193 @@
+package ibm
+
+import (
+	"context"
+	"strings"
+
+	"github.com/IBM/networking-go-sdk/dnsrecordsv1"
+	"github.com/IBM/networking-go-sdk/globalloadbalancerv1"
+	"github.com/IBM/networking-go-sdk/zonessettingsv1"
+	"github.com/IBM/networking-go-sdk/zonesv1"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableIbmCISDomain(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "ibm_cis_domain",
+		Description: "IBM CIS Domain",
+		List: &plugin.ListConfig{
+			Hydrate: listCISDomains,
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getCISDomain,
+			KeyColumns: plugin.SingleColumn("id"),
+		},
+		Columns: []*plugin.Column{
+			// Top columns
+			{Name: "id", Type: proto.ColumnType_STRING, Description: "The zone id."},
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "The zone name."},
+
+			// Other columns
+			{Name: "account_id", Type: proto.ColumnType_STRING, Hydrate: getAccountId, Description: "An unique ID of the account.", Transform: transform.FromValue()},
+			{Name: "created_on", Type: proto.ColumnType_TIMESTAMP, Description: "The date and time that the zone was created."},
+			{Name: "modified_on", Type: proto.ColumnType_TIMESTAMP, Description: "The date and time that the zone was updated."},
+			{Name: "minimum_tls_version", Type: proto.ColumnType_STRING, Hydrate: getTlsMinimumVersion, Description: "The tls version of the zone.", Transform: transform.FromField("Value")},
+			{Name: "original_registrar", Type: proto.ColumnType_STRING, Description: "The original registrar of the zone."},
+			{Name: "original_dnshost", Type: proto.ColumnType_STRING, Description: "The original dns host of the zone."},
+			{Name: "status", Type: proto.ColumnType_STRING, Description: "The zone status."},
+			{Name: "paused", Type: proto.ColumnType_BOOL, Description: "Whether the zone is in paused state."},
+			{Name: "web_application_firewall", Type: proto.ColumnType_STRING, Hydrate: getWebApplicationFirewall, Description: "The web application firewall status.", Transform: transform.FromField("Value")},
```

Comment:
Should the name of the column be `web_application_firewall` since the description refers to the status?